frontend/ic_vetkeys/src/utils/utils.ts

```diff
@@ -25,29 +25,24 @@ export class TransportSecretKey {
     readonly #pk: G1Point;
 
     /**
-     * Construct a new TransportSecretKey from a bytestring
-     *
-     * The bytestring should be randomly generated by a cryptographically
-     * secure random number generator.
+     * Create a random transport secret key
+     */
+    static random() {
+        return new TransportSecretKey(bls12_381.utils.randomPrivateKey());
+    }
+
+    /**
+     * Deserialize TransportSecretKey from a bytestring
      *
-     * For most applications, prefer using the static method random
+     * The passed value would typically be a string previously returned
+     * by calling getSecretKey on a randomly-created TransportSecretKey.
```

Comment:
In Rust, we have `TransportSecretKey::serialize` while here in TS it is `TransportSecretKey.getSecretKey`. Not sure if it is intended to also unify this.

Reply:
Probably makes sense to use serialize/deserialize always